doc/CMakeLists.txt

```diff
@@ -27,7 +26,12 @@ if(SFML_OS_WINDOWS)
                  PATHS "C:/Program Files/HTML Help Workshop" "C:/Program Files (x86)/HTML Help Workshop"
                  DOC "HTML Help Compiler program")
     if(DOXYGEN_HHC_PROGRAM)
-        set(DOXYGEN_GENERATE_HTMLHELP YES)
+        if(DOXYGEN_VERSION LESS "1.10.0")
+            set(DOXYGEN_GENERATE_HTMLHELP YES)
+        else()
+            message("Due to conflicts with the HTML output settings in Doxygen ${DOXYGEN_VERSION}, the HTML Help generation will be disabled")
+            set(DOXYGEN_GENERATE_HTMLHELP NO)
```

Comment:
Nit: We tend to use either `TRUE`/`FALSE` or `ON`/`OFF` for CMake booleans. CMake accepts many different forms but those two are what we try to stick to.

Reply:
Yes, except this is a doxygen flag and not a CMake flag. 😉 

![image](https://github.com/SFML/SFML/assets/920861/1b97f845-60b4-4345-8836-35002c84ca1c)

Reply:
![image](https://github.com/SFML/SFML/assets/39244355/950ee9f4-1c3e-4f2d-b367-69fb21dafa9b)